src/main/java/segelzwerg/sporttooolbox/IUnits/MinutesPerHundredMeters.java

```diff
@@ -0,0 +1,37 @@
+package segelzwerg.sporttooolbox.IUnits;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Pace of minutes per hundred meters
+ */
+@Getter
+@Setter
```

Comment:
@GledsonAfonso for the lot of work you put into it. I think for the moment we don't need getters and setters, because we can test equality with
`Pace expectedPace = new MinutesPerKilometer(61.05f);`
`Pace pace = new MinutesPerHundredMeters(6.105f).toMinutesPerKilometer();`
`assertThat(expectedPace, equaltTo(pace)`

Reply:
Okay, thanks for the feedback! Annotations already removed and review re-requested.